specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/errored.class/request.rpt

```diff
@@ -0,0 +1,11 @@
+#YWRtaW46YWRtaW4= : admin:admin   dGVzdDp0ZXN0 : test:test   am9lOndlbGNvbWU= : joe:welcome (base64)
+
+connect http://localhost:8000/snmp
+connected
+
+write method "GET"

```

Comment:
It is worth putting a comment that this is an *invalid* WebSocket request in case it isn't clear to a new dev.

This test really isn't applicable to RFC 7235 - we should be testing http only in these spec tests.